Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -231,32 +231,23 @@ extension TransactionHistoryViewController: UITableViewDataSource, UITableViewDe
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = self.tableView.dequeueReusableCell(withIdentifier: TransactionTableViewCell.identifier) as! TransactionTableViewCell
-        if self.Filtercontent.count != 0 {
-            let transaction = Filtercontent[indexPath.row]
-            cell.setData(transaction)
+        guard filteredTransactions.count != 0 else {
```

Comment:
`if self.filteredTransactions.count == 0 {` would be easier to read I believe.

Reply:
if self.filteredTransactions.isEmpty is even better.

Reply:
fixed in 28f46d1